JS/Lesson32/onur/my-app/src/components/RecipeList/index.jsx

```diff
@@ -1,12 +1,31 @@
-export const RecipeList = ({recipes}) => {
+import { useState } from "react";
+import { RecipeDetail } from "../RecipeDetail";
+import './styles.css'
+
+
+export const RecipeList = ({ recipes }) => {
+
+	const [recipe, setRecipe] = useState(false);
+	const [SelectedRecipe, setSelectedRecipe] = useState(null);
+
+	const selectRecipe = (recipe) => {
+		setRecipe(true);
+		setSelectedRecipe(recipe);
+	}
+
 	return (
-		<div>
-			<h2>Search Results</h2>
-			<ul>
-				{recipes.map((recipe) => {
-					return <li key={recipe.id}>{recipe.strMeal}</li>
-				})}
-			</ul>
+		<div className="list-container">
+			<div className="main">
+				<h2>Search Results</h2>
```

Comment:
I think using "Recipes" in the headline would make more sense in the layout that you chose:
```suggestion
				<h2>Recipes</h2>
```